main.js

```diff
@@ -11,13 +11,15 @@ const BOARD_SIZE = 14;
 const HALF_BOARD_SIZE = BOARD_SIZE / 2;
 const SPHERE_RADIUS = 0.4;
 const SPHERE_WIDTH_SEGMENT = 32;
-const SPHERE_HEIGHT_SEGMENT = SPHERE_WIDTH_SEGMENT / 2;
+const SPHERE_HEIGHT_SEGMENT = SPHERE_WIDTH_SEGMENT / 3;
 
 const scene = new THREE.Scene();
 let currentPlayer = 'black';
+let currentPlayerLabel = null;
+let restartButtonLabel = null;
 
 function initializeGameBoard() {
-    const gameBoard = Array(BOARD_SIZE).fill().map(() => Array(BOARD_SIZE).fill(null));
+    const gameBoard = Array(BOARD_SIZE + 1).fill().map(() => Array(BOARD_SIZE + 1).fill(null));
     return {gameBoard};
```

Comment:
Not anything wrong here, but just wondering why you need to wrap this in an object?

Reply:
gameBoard get's used later on in main().